.github/workflows/ci.yml

```
@@ -0,0 +1,67 @@
+---
+name: ci
+
+on:
```

This removes this warning.
```suggestion
"on":
```

---

Cool, I have that in a bunch of places and no idea what it meant.